CloudFoundry/content/apps/envvar.md

```diff
@@ -0,0 +1,42 @@
+---
+menu:
+  main:
+    parent: apps
+title: Environment Variables
+weight: -1
+---
+
+Environment variables are the means by which the Cloud Foundry runtime communicates with a deployed application about its environment. 
+
+## Setting environment variables in the manifest file
+
+You can set environment variables in your app by adding an `env` block to the manifest file. The `env` block consists of a heading, then one or more environment variable/value pairs.
```

Comment:
I'd suggest a slight rewording here for improved clarity...
You can set environment variables within your app containers by adding an `env` block to the manifest.yml file for your app.